lib-multisrc/heancms/src/eu/kanade/tachiyomi/multisrc/heancms/HeanCms.kt

```diff
@@ -70,6 +76,39 @@ abstract class HeanCms(
         .add("Origin", baseUrl)
         .add("Referer", "$baseUrl/")
 
+    private fun authHeaders(): Headers {
+        val builder = headersBuilder()
+        if (preferences.loginPref && enableLogin) {
```

Comment:
```suggestion
        if (enableLogin && preferences.loginPref) {
```
check local value first before hitting the preference